content/blog/monitor-application-performance-across-hybrid-cloud-environment-using-apache-skywalking-and-service-mesh.md

```diff
@@ -0,0 +1,331 @@
+service/istio-egressgateway    ClusterIP      172.20.224.42   <none>                                                                    80/TCP,443/TCP                                                               38d
+service/istio-ingressgateway   LoadBalancer   172.20.249.41   a17641fd9b6564b02ab3cc5faeb51e7a-1241343104.us-east-2.elb.amazonaws.com   15021:30506/TCP,80:31341/TCP,443:31933/TCP,31400:31131/TCP,15443:32118/TCP   38d
+service/istiod                 ClusterIP      172.20.95.232   <none>                                                                    15010/TCP,15012/TCP,443/TCP,15014/TCP                                        38d
+
+NAME                                   READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/istio-egressgateway    1/1     1            1           38d
+deployment.apps/istio-ingressgateway   1/1     1            1           38d
+deployment.apps/istiod                 1/1     1            1           38d
+
+NAME                                              DESIRED   CURRENT   READY   AGE
+replicaset.apps/istio-egressgateway-78fb5cf46     1         1         1       38d
+replicaset.apps/istio-ingressgateway-77b9d69b74   1         1         1       38d
+replicaset.apps/istiod-67fcb675b5                 1         1         1       38d
+```
+
+### Deploy Online Boutique application
+
+Please follow up the deployment process in my blog post [here](https://developer.hpe.com/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise/) to deploy the Online Boutique application across the public AWS EKS cluster and the private Kubernetes cluster in HPE GreenLake for Private Cloud Enterprise. The *Skupper* CLI tool is used for deploying the Virtual Application Network (VAN) and creating connection between private Kubernetes cluster and public AWS EKS cluster.
+
+After deployed the Online Boutique application, you can verify deployment by checking the *Skupper* status from the deployment namespace *boutique*: 
+
+* from private Kubernetes cluster:
+
+```markdown
+$ skupper status --namespace boutique
+Skupper is enabled for namespace "boutique" with site name "pce-private" in interior mode. 
+It is connected to 1 other site. It has 10 exposed services.
+```
+
+* from AWS EKS cluster:
+
+```markdown
+$ skupper status --namespace boutique
+Skupper is enabled for namespace "boutique" with site name "aws-public" in interior mode. 
+It is connected to 1 other site. It has 10 exposed services.
+The site console url is: https://aea867abf6fb6413d8f577652da564c1-130946084.us-east-2.elb.amazonaws.com:8080
+The credentials for internal console-auth mode are held in secret: 'skupper-console-users‘
+```
+
+The Skupper console shows the connections from the public AWS EKS cluster to the private Kubernetes cluster and the hybrid deployment of the Online Boutique application:
+
+![](/img/skupper-apps.png)
```

Comment:
Denis told it could be the same image name being used in both the first blog and this second one. I can double check it when the blog blog gets published. Otherwise, I can re-upload the image with a different name.

Reply:
We may need to fix these post publication. I still couldn't see them.